### PR TITLE
pass params to report

### PR DESCRIPTION
**Pull request by taboularasa**
@patbenatar I would like to filter my collection. the easiest way would be pass down the params hash. can you think of alternatives?

### CODE REVIEW

File: app/workers/report_card/runner.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ module ReportCard
   class Runner
     include Sidekiq::Worker
 
-    def perform(klass_name, recipient_email)
-      report = ReportCard::Report.find(klass_name).new
+    def perform(params, recipient_email)
+      report = ReportCard::Report.find(params[:report_name]).new(params)
```
Comment (patbenatar) [inline app/workers/report_card/runner.rb]: Could remove `report_name` from `params` before passing to initializer.. no longer needed

Comment (patbenatar) [inline app/workers/report_card/runner.rb]: Not sure if worth it though

**Comment by patbenatar** [comment]
@taboularasa see comment that got hidden on outdated diff

**Comment by taboularasa** [comment]
@patbenatar I vote not to pop :report_name off the params hash, not worth it

**Comment by patbenatar** [comment]
@taboularasa Agreed. :shipit:

**Pull request merged**